schemas/fingerprint-server-api.yaml

```diff
@@ -1132,6 +1136,8 @@ components:
                     • `true` - Emulated environment detected (e.g. launch inside of AVD)
                     • `false` - No signs of emulated environment detected or the client is not Android.
                   example: false
+              required:
```

Comment:
@ilfa Would it also make sense to set `additionalProperties: false` for each `{product}.data` object? 

Then CI tests could check if unexpected/undocumented properties appear in live responses.

Reply:
Gotcha, thanks!

Reply:
Wait, would SDKs actually break if they encountered a new property in the response? Wouldn't they just ignore it? Or it would be inaccessible in the created Response object/Class? Not questioning the decisions, just making sure I understand :)

Reply:
What about `additionalProperties: false` directly on `{product}`? It seems `botd` and `identification` already have it, but other products don't. Should it be unified one way or another?

Reply:
I rechecked and actually, it doesn't affect our SDKs.
We fixed this problem with other instruments, so I can add it to the all signals.

Reply:
fixed!